### PR TITLE
[Change] - Made customStartup_* scheduled.

### LINKED ISSUE #114: [Bug] customStartup_* should be wrapped in a spawn block
**Describe the bug**
It isn't obvious that you need to bring your own spawn block into the customStartup_* scripts, JC's mission today didn't add it.  Therefore half of the framework startup ends up waiting for the custom startup code to finish.  lmao

**To Reproduce**
1. go to CA server
2. load up `CA_CO30_JC_OpSUTTON_v3`
3. checkem

**Expected behavior**
Users can decide whether the startup code needs to be sync or async.  It's still important for the startup code to be able to be sync for certain scenarios.

**Screenshots**
lol no

**Environmental Info(please complete the following information):**
 - Main repo
 - No cdlc
 - Framework Version 1.4.1

**Additional context**
Solve this by adding in a spawn block by default, and a comment above it explaining what it's doing there.

### PR DESCRIPTION
**Pull request by Bubbus**
### Pull Request Description
**When merged this pull request will:**
- Makes customStartup_* scheduled.  
- Adds documentation into those files to describe what this means.

### Release Notes
Made customStartup_* scheduled.  Added documentation into those files to describe what this means.

## IMPORTANT

- [x] Testing has been completed as neccessary, depending on the nature & impact of the changes.
- [x] The Release Notes section below must be filled out appropriately to explain the changes made in this PR. This section will be used in Framework Changelogs.
- [x] If the contribution affects [the wiki](https://github.com/CombinedArmsGaming/CAFE3/wiki), please include your changes in this pull request so the wiki is consistently updated.
- [x] [Contribution Guidelines](https://github.com/CombinedArmsGaming/CAFE3/blob/release/contributing.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `[Descriptor] - Add|Fix|Improve|Change|Make|Remove {changes}`.

Resolves #114

### CODE REVIEW

File: customStartup_client.sqf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Use this file to run code when the mission starts, or a player joins.
+
+// This file runs client-side, so code here will run for every player.  If there are 20 players, it will run 20 times (1 each)!
```
Comment (Cre8or) [inline customStartup_client.sqf]: I understand where this is coming from, but I don't fully agree with the wording - let me explain:

The file is executed once *on every machine*. This means, the client's machine executes it *once* - irrespective of how many players are on the server.

The comment describes a special case: the usage of code with global effect. In that case, the global effect will be reapplied for every connected (and connecting) player. However, I don't think it bears mentioning, because this file is intended for clientside usage only (which inherently forbids anything with global effect). Anything else should go into the *serverside* init file (that's why it exists).

**tl;dr:** valid point, but misleading phrasing.

**Review by Cre8or — COMMENTED** [review]
Thoughts on how to improve the wording? (re: my comment on the client startup file)

File: customStartup_client.sqf (hunk 1 above)
Comment (Bubbus) [reply customStartup_client.sqf]: I want a warning for this kind of stuff because locality is a thing even I'm still screwing up all the time, and MMs aren't always programmers.  But yeah I'm not wedded to the wording, if anyone's got clarity suggestions we can go with those.

**Review by dgibso29 — APPROVED** [review]
☕

**Review by Cre8or — APPROVED** [review]
Yep, that's better. Honestly can't stress the "player-specific" bit enough, as it's the biggest gotcha in ArmA multiplayer scripting imo.

🚢 it
